### ISSUE TITLE
Question about running the pretrain.py

### ISSUE THREAD

**Issue by susantiyuni** [dhlee347/pytorchic-bert#23]
Hey,

I have difficulties in running the pretrain, any help would be appreciated.
So I've prepared corpus.txt (quite small, about 1000 lines) that looks like this:

```
document 1 line 1...
document 1 line 2...
document 1 line 3...

document 2 line 1...
document 2 line 2...
document 2 line 3...
```

And I run the pretrain.py but I got an error on train.py file, on this line:
`print('Epoch %d/%d : Average Loss %5.3f'%(e+1, self.cfg.n_epochs, loss_sum/(i+1)))`
So for the time being I commented that line. 

And after I run again, here what I got:
```
Iter (loss=X.XXX): 0it [00:00, ?it/s]
Iter (loss=X.XXX): 0it [00:00, ?it/s]
Iter (loss=X.XXX): 0it [00:00, ?it/s]
Iter (loss=X.XXX): 0it [00:00, ?it/s]
Iter (loss=X.XXX): 0it [00:00, ?it/s]
Iter (loss=X.XXX): 0it [00:00, ?it/s]
Iter (loss=X.XXX): 0it [00:00, ?it/s]
Iter (loss=X.XXX): 0it [00:00, ?it/s]
Iter (loss=X.XXX): 0it [00:00, ?it/s]
....
```

Could you please point me where I could possibly make the mistake?
Thanks!

p.s. I have commented some part of the code in train.py (the part where it loads the checkpoint, because I dont install the tensorflow for a reason). What I want to do for now is training a pretrained bert model using my own data. I am not sure if it is causing the error above?

**Comment by Seoinsuk1** [comment]
same here

**Comment by Tikquuss** [comment]
I had the same problem as you.
But after I had checked I noticed that my dataset was empty (I'm talking about `data_iter` on line [`219`](https://github.com/dhlee347/pytorchic-bert/blob/master/pretrain.py#L219) of [`pretrain.py`](https://github.com/dhlee347/pytorchic-bert/blob/master/pretrain.py)), because I specified the data in the wrong format.

My problem has been solved.

![bt](https://user-images.githubusercontent.com/49171640/107893454-9fc9a000-6f2b-11eb-84e5-49289a164391.png)
